helm/templates/pvc-config.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if not .Values.authelia.ldap.enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "authelia.fullname" . }}-etc
```

Comment:
the name could be more explicit than `-etc` I guess. We are defining this claim for the users database if I'm not mistaken so I'd rather call it something like `{{ template "authelia.fullname" . }}-userdb`. What do you think?

Reply:
Actually I'm placing the user database in /var/lib/authelia as of now. But that can change, and I'd rather call it -config. This is purely internal anyway, the user never touches or changes this

Reply:
I'm still a bit confused because there is no usage of this claim in the rest of the Chart unlike the "-store" claim. Is it expected?

Reply:
Yeah, this was made when I thought I could do a cool trick with the userlists. It's gone now.